components/opponent/wikis/rocketleague/player_ext_custom.lua

```diff
@@ -32,7 +32,7 @@ function PlayerExtCustom._fetchTeamFromPlacement(resolvedPageName, date)
 	local conditions = {
 		'[[opponenttype::solo]]', -- can not use Opponent.solo due to circular requires
 		'[[pagename::' .. mw.title.getCurrentTitle().text:gsub(' ', '_') .. ']]',
-		'[[opponentname::' .. resolvedPageName .. ']]',
+		'[[opponentname::' .. resolvedPageName:gsub('_', ' ') .. ']]',
```

Comment:
possibly (to make it future proof for when we gradually switch it to underscores)
```suggestion
		'([[opponentname::' .. resolvedPageName .. ']] OR [[opponentname::' .. resolvedPageName:gsub('_', ' ') .. ']])',
```